crates/iroha_data_model/src/query/mod.rs

```diff
@@ -746,12 +746,12 @@ pub mod asset {
         #[ffi_type]
         pub struct FindAssets;
 
-        /// [`FindAssetsDefinitions`] Iroha Query finds all [`AssetDefinition`]s presented
+        /// [`FindAllAssetDefinitions`] Iroha Query finds all [`AssetDefinition`]s presented
         /// in Iroha Peer.
         #[derive(Copy, Display)]
         #[display(fmt = "Find all asset definitions")]
         #[ffi_type]
-        pub struct FindAssetsDefinitions; // TODO: Should it be renamed to [`FindAllAssetDefinitions`?
+        pub struct FindAllAssetDefinitions;
```

Comment:
I think this should be left as `FindAssetsDefinitions`, and the `TODO` removed

Reply:
You're right, updated.